app/services/report_importer.rb

```diff
@@ -50,12 +50,21 @@ def import
     end
   end
 
-  def scan
+  def add_reporter_specific_data
     logger.info "Scanning report with: #{report_scanners.join(', ')}"
     report_scanners.each do |scanner|
-      break if scanner.scan(report, logs)
+      if scanner.respond_to?(:identify_origin)
```

Comment:
Since the scanner is supposed to implement both `identify_origin` and `add_reporter_data`, maybe this should be

 `if scanner.respond_to?(:identify_origin) && scanner.respond_to?(:add_reporter_data)`